homeassistant/components/serial_pm/sensor.py

```diff
@@ -3,15 +3,15 @@
 
 import voluptuous as vol
 
+from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import CONF_NAME
-from homeassistant.helpers.entity import Entity
 import homeassistant.helpers.config_validation as cv
-from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
```

Comment:
Please do not mix formatting changes with code changes.

Reply:
Well, we should thankful that Fab do that work and don't add more burden to it.